Modelica_StateGraph2/package.mo

```diff
@@ -11844,7 +11844,11 @@ This is the same model as
           smooth=Smooth.Bezier));
       annotation (experiment(StopTime=2),
         Diagram(coordinateSystem(
-              preserveAspectRatio=true, extent={{-100,-100},{100,100}})));
+              preserveAspectRatio=true, extent={{-100,-100},{100,100}})),
+        Documentation(info="<html>
+This model may contain a logic error.
+It will translate and simulate without errors and <a href=\\\"modelica:/Modelica_StateGraph2.Examples.Applications.HarelsWristWatch.WristWatch\\\">WristWatch</a> has similar constructs.

```

Comment:
```suggestion
However, it will translate and simulate without errors and <a href=\\\"modelica:/Modelica_StateGraph2.Examples.Applications.HarelsWristWatch.WristWatch\\\">WristWatch</a> has similar constructs.

```